package Exception_Handeling;

import java.util.Scanner;

public class Try_Catch {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		int x,y,z;
		Scanner sc =new Scanner(System.in);
		System.out.println("Enter the  x");
		x=sc.nextInt();
		System.out.println("Enter the y");
		y=sc.nextInt();
		z= (x/y);
		

	}

}
